applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -438,6 +442,89 @@ static size_t cmd_name_toupper(char *cmd, size_t cmd_len)
 	return i;
 }
 
+static void restore_at_backend(void)
+{
+	const int err = at_backend.start();
+
+	if (err) {
+		LOG_ERR("Failed to restore AT backend. (%d) Resetting.", err);
+		slm_reset();
+	}
+}
+
+static int stop_at_backend(void)
+{
+	const int err = at_backend.stop();
+
+	if (!err) {
+		/* Wait for UART disabling to complete. */
+		k_sleep(K_MSEC(100));
+	}
+	return err;
+}
+
+int slm_at_set_backend(const struct slm_at_backend new_backend)
```

Comment:
You mean from CMUX? This is correct.
I did experiment with this possibility, but faced issues where the application would just hang and so left it out. I don't remember exactly, but possibly found how to do it at some point, so it may be possible to add at a later point if judged useful (currently, `AT#XRESET` does the job).
(Also, I feel like the 3GPP standard for the CMUX command does not define how to stop it. I wonder whether modems supporting CMUX do support stopping it.)